lib/uktest/include/uk/test.h

```diff
@@ -668,6 +684,49 @@ _uk_test_do_assert(struct uk_testcase *esac, unsigned short line, int cond,
 	}
 }
 
+static void __maybe_unused
+_uk_fixme_do_assert(struct uk_testcase *esac, unsigned short line, int cond,
+		   const char *fmt, ...)
+{
+	struct uk_assert *assert;
+#ifdef CONFIG_LIBUKTEST_LOG_TESTS
+	va_list ap;
+	int pad_len;
+	char msg[UKT_COLWIDTH];
+
+	/* Save formatted message to buffer. */
+	va_start(ap, fmt);
+	vsnprintf(msg, sizeof(msg), fmt, ap);
+	va_end(ap);
+
+	/* Output the result. */
+	pad_len = UKT_COLWIDTH - strlen(msg);
+	UK_ASSERT(pad_len >= 0);
+
+#ifndef CONFIG_LIBUKTEST_STRICT_MODE
+	uk_test_printf("%s %*.*s %s\n",
+				msg,
+				pad_len, pad_len,
+				UKT_PADDING,
+				UKT_OMITTED);
+#else
+    uk_test_printf("%s %*.*s %s\n",
+				msg,
+				pad_len, pad_len,
+				UKT_PADDING,
+				cond ? UKT_OMITTED_PASSED : UKT_OMITTED_FAILED);
+#endif
+#endif
+	assert = _uk_test_find_assert(esac, line);
+
+	UK_ASSERT(assert);
+
+	if (cond)
+		assert->status = UK_TEST_ASSERT_OMITTED_SUCCESS;
+	else
+		assert->status = UK_TEST_ASSERT_OMITTED_FAIL;
```

Comment:
Is strict mode supposed to treat fixmes as fails? If so, should we handle the case when both strict mode and `FAILFAST` are enabled? As in fail on first FIXME fail.